ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/ExpenseItemDetailView.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Activity that lets the user view 
+ * information of the selected ExpenseItem
+ * 
+ * Covers Issue 16
+ * @author Oleg Oleynikov 
+ * @version 1.0
+ * @since 2015-03-15
+ */
+
+package ca.ualberta.cs.shinyexpensetracker;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaim;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaimList;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaimsRepository;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseItem;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+
+public class ExpenseItemDetailView extends Activity {
+
+	private ExpenseItem item;
+	private Context context = ExpenseItemDetailView.this;
+	
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_expense_item_detail_view);
+		
+		Intent intent = getIntent();
+		Bundle bundle = intent.getExtras();
+		
+		if (bundle != null){
+			int claimId = (Integer) bundle.get("Claim ID");
+			int expenseItemId = (Integer) bundle.get("Item ID");
+			ExpenseClaimController controller = ExpenseClaimController.getInstance();
+			ExpenseClaim claim = controller.getExpenseClaim(claimId);
+			item = claim.getItemById(expenseItemId);
+			populateTextViews();
+		}
+		// this one is just for testing purposes
```

Comment:
I assume you're done testing this, so feel free to delete this commented-out code now.